docs/credentials.md

```diff
@@ -130,14 +130,20 @@ svc <- paws::svc(
 
 `expiration` must be a `POSIXct` date-time or able to be compared with them.
 
+Additionally, anonymous credentials may be used (e.g., before calling `assume_role_with_web_identity()` on an STS service)
+``` r
+svc <- paws::svc(config = list(credentials = list(anonymous = TRUE)))
```

Comment:
This fully unblocked me once I stumbled upon the great discussion in #512, so thought it'd be helpful to pull out more visibly!